webapp/components/wpt-amend-metadata.js

```diff
@@ -361,6 +365,9 @@ class AmendMetadata extends LoadingState(PathInfo(ProductInfo(PolymerElement)))
       .then(text => {
         this.prLink = text;
         this.prText = 'Created PR: ' + text;
+        // Create an observeable change to trigger /api/metadata/pending
+        // reload.
+        this.triageObserver = !this.triageObserver;
```

Comment:
If what we're trying to do is inform some ancestor of ours that an event has occurred (metadata being triaged), we should use dispatchEvent with a `computed=true, bubbles=true` event. Our ancestor can then have an event listener that will receive the events, and not have to hack this two-way binding all the way down!

See:

https://polymer-library.polymer-project.org/3.0/docs/devguide/events#custom-events
https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/dispatchEvent